modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_globan\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * The form for managing the configuration of the Webtools Globan module.
+ */
+class WebtoolsGlobanSettingsForm extends ConfigFormBase {
+
+  /**
+   * The language manager.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
+  /**
+   * Constructs a WebtoolsGlobanSettingsForm object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The factory for configuration objects.
+   * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
+   *   The Language Manager service.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, LanguageManagerInterface $language_manager) {
+    parent::__construct($config_factory);
+    $this->languageManager = $language_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('config.factory'),
+      $container->get('language_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'oe_webtools_globan_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('oe_webtools_globan.settings');
+
+    $form['globan_settings'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Global banner settings'),
+    ];
+    $form['globan_settings']['livepreview'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Check appearance'),
+      '#markup' => '<div id="globan-preview"></div>',
+      '#description' => $this->t('Link for testing appearance of global banner.'),
+    ];
+    $form['globan_settings']['display_eu_flag'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Display EU flag'),
+      '#options' => [
+        '0' => $this->t('No - hide flag'),
```

Comment:
This should be int and in the .schema it should be a boolean not string. It's a flag. Same for all the other flags.